StRoot/StarGenerator/FILT/FcsDYBGFilterSingle.cxx

```diff
@@ -0,0 +1,89 @@
+#include "FcsDYBGFilterSingle.h"
+
+#include "StarGenerator/EVENT/StarGenParticle.h"
+#include "StarGenerator/EVENT/StarGenEvent.h"
+#include <string>
+#include <iostream>
+#include <fstream>
+#include <cmath>
+#include <vector>
+#include <algorithm>
+
+static const float ZFCS    = 710.16 + 13.9 + 15.0;
```

Comment:
Not critical, but you should consider making these parameters tunable (i.e. non-const, loadable from a configuration file...)